.github/workflows/content-release.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         continue-on-error: true
         with:
           payload: '{"attachments": [{"color": "#2EB67D","blocks": [{"type": "section","text": {"type": "mrkdwn","text": "Stand by, content release for next-build coming up (using ${{ needs.validate-build-status.outputs.TAG }}). <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}>"}}]}]}'
-          channel_id: ${{ env.SLACK_CHANNEL }}
+          channel_id: C06DSBT7CBW # hard code for now
```

Comment:
This scares me a bit. Curious to why we are hard coding it vs grabbing it via env. Also please update the PR to include the ticket number and testing steps

Reply:
Good catch, that was a copy/paste error.